x/ecocredit/server/core/utils.go

```diff
@@ -0,0 +1,19 @@
+package core
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+)
+
+// getCreditType searches for a credit type that matches the given abbreviation within a credit type slice.
+func (k Keeper) getCreditType(ctAbbrev string, creditTypes []*ecocredit.CreditType) (ecocredit.CreditType, error) {
```

Comment:
Let's make sure there's an issue to track migrating credit types to ORM and out of params

Reply:
https://github.com/regen-network/regen-ledger/issues/729